src/components/Tooltip.tsx

```diff
@@ -0,0 +1,49 @@
+import { BiSolidHelpCircle } from "solid-icons/bi";
+
+import { useGlobalContext } from "../context/Global";
+import "../style/tooltip.scss";
+
+const Tooltip = ({ label }) => {
+    const TIMEOUT = 300;
+    const TIMEOUT_CLICK = 1000;
+
+    const { t } = useGlobalContext();
+
+    let timeout: any = null;
+
+    const tooltipClick = (evt: MouseEvent) => {
+        const target = evt.currentTarget as HTMLSpanElement;
+        (target.parentNode as HTMLSpanElement).classList.add("active");
+        timeout = setTimeout(() => {
+            (target.parentNode as HTMLSpanElement).classList.remove("active");
+        }, TIMEOUT_CLICK);
+        evt.stopPropagation();
+    };
```

Comment:
feels a bit unnecessary

Reply:
its for mobile devices, hover wont work well there